src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.io.IOException;
+
+public class Duke {
+
+    protected static ArrayList<Task> list;
+    public static void main(String[] args) throws DukeException {
+        Ui.welcome();
+
+        try {
+            list = LoadFile.load();
+
+            if (!LoadFile.loaded) {
+                list = new ArrayList<>();
+            }
+
+            Parser.start();
+
+        } catch (DukeException e) {
```

Comment:
Maybe you can use 'catch (Exception1 | Exception2 ex)' to catch two exceptions at once